Trimble.Modus.Components/Enums/SwitchLabelPosition.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Trimble.Modus.Components.Enums;
+
+public enum SwitchLabelPosition
```

Comment:
Can you make it generic? Like Label or TitlePosition? We can use it in other controls too in the future if necessary

Reply:
Sure